ledger.go

```diff
@@ -0,0 +1,61 @@
+package apihttp
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	pcrypto "github.com/libp2p/go-libp2p-core/crypto"
+	xcrypto "github.com/proximax-storage/go-xpx-crypto"
+	drive "github.com/proximax-storage/go-xpx-dfms-drive"
+)
+
+type apiLedger apiHttp
+
+func (api *apiLedger) ListDrives(ctx context.Context) ([]drive.ID, error) {
```

Comment:
You need to implement the new command in `go-xpx-dfms`, (e.g. `ledger/drives`) and use it.

And look at `apiContractClient.List()`, it has similar goal.